crates/transaction-pool/src/validate/eth.rs

```diff
@@ -61,6 +61,62 @@ impl<Client, Tx> EthTransactionValidator<Client, Tx> {
     pub fn client(&self) -> &Client {
         &self.inner.client
     }
+
+    /// Returns the tracks activated forks relevant for transaction validation
+    pub fn fork_tracker(&self) -> &ForkTracker {
+        &self.inner.fork_tracker
+    }
+
+    /// Returns if there are EIP-2718 type transactions
+    pub fn eip2718(&self) -> bool {
+        self.inner.eip2718
+    }
+
+    /// Returns if there are EIP-1559 type transactions
+    pub fn eip1559(&self) -> bool {
+        self.inner.eip1559
+    }
+
+    /// Returns if there are EIP-4844 blob transactions
+    pub fn eip4844(&self) -> bool {
+        self.inner.eip4844
+    }
+
+    /// Returns if there are EIP-7702 type transactions
+    pub fn eip7702(&self) -> bool {
+        self.inner.eip7702
+    }
+
+    /// Returns the current max gas limit
+    pub fn block_gas_limit(&self) -> &AtomicU64 {
```

Comment:
we can call this function here

https://github.com/paradigmxyz/reth/blob/cf8ff9829cca8a16bbed99a2de9e3d85b8dfe92b/crates/transaction-pool/src/validate/eth.rs#L674-L676

and return u64